05/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>uklad-strony</title>
+    <link
+      rel="stylesheet"
+      href="./style/reset.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style/base.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./style/index.css"
+    />
+    <link
+      rel="stylesheet"
+      media="(max-width: 768px)"
+      href="./style/mobile.css"
+    />
```

Comment:
Jeśli robimy mobile first to powinniśmy wczytaywać taki plik w pierwszej kolejności i bez warunkowania (media)

Reply:
Dziękuję za uwagi ;) BEM, zmienne oraz mobile first stosuję już w projekcie. Natomiast zadanie 5 nie robiłam mobile first, nie wiem czemu, gdzieś się zagapiłam.